taas_api/data.py

```diff
@@ -113,6 +114,10 @@ def validate(self):
             if not (0 <= self.schedule_discretion <= 1):
                 return False, ["schedule_discretion out of range, must be [0,1]"]
 
+        if self.exposure_tolerance is not None:
+            if not (0.1 <= self.exposure_tolerance <= 1):
```

Comment:
nit: and instead of nested if

Reply:
most of these checks for optional params are nested ifs, should i change them all?